_pytest/deprecated.py

```diff
@@ -22,3 +27,6 @@
 GETFUNCARGVALUE = "use of getfuncargvalue is deprecated, use getfixturevalue"
 
 RESULT_LOG = '--result-log is deprecated and scheduled for removal in pytest 4.0'
+
+MARK_INFO_ATTRIBUTE = RemovedInPytest4_0Warning(
+    "Markinfo attributes are deprecated, please iterate the mark Collection")
```

Comment:
This should be documented on the docs, I think. Users who see this probably won't know what this means.

I also suggest to add a "(see <link>)" pointing to the docs so users can easily go to it to know how to update their code.